.github/workflows/practica-quality-check.yml

```diff
@@ -31,8 +31,12 @@ jobs:
 
     - run: npm ci
 
-    - name: Generate coverage report
-      run: npm test -- --coverage
+    - name: Run tests and generate coverage report
```

Comment:
could you please disable the code coverage report as we already have codecov 😄 

for more how to do it - [`mattallty/jest-github-action` - Silencing the code coverage comment](https://github.com/mattallty/jest-github-action#silencing-the-code-coverage-comment)

Reply:
Yea, this would result in abmiguity

Reply:
I removed the coverage comment